### PR TITLE
Replace webkit theming design by BBj default theming

### PR DESCRIPTION
**Pull request by saadnoor**
This pull request: 
- Removes all webkit variables with bbj variables
- Fix the bug regarding loading theme files

### CODE REVIEW

File: framework/LoginDialog/LoginDialog.css [hunk 1]
```diff
@@ -50,11 +50,14 @@
 
 .loginWidgetBtnStyle{
     grid-row-start: 5;
-    --bbj-color-default: var(--webkit-primary);
     height: 43px;
     width: 100%;
 }
 
+/* .loginWidgetBtnStyle:hover{
```
Comment (StephanWald) [inline framework/LoginDialog/LoginDialog.css:57]: do we want to remove this comment?

**Review by StephanWald — APPROVED** [review]
maybe remove that one comment (I left a remark) if it can be erased

**Review by trifiasco — APPROVED** [review]
looks good :+1:

File: framework/LoginDialog/LoginDialog.css (hunk 1 above)
Comment (saadnoor) [reply framework/LoginDialog/LoginDialog.css:57]: Thanks for noticing, removed

**Pull request merged**